source/develop/release-management/features/network/ovirt-lldp-labeler.html.md

```diff
@@ -0,0 +1,155 @@
+---
+title: oVirt LLDP labeler
+category: feature
+authors: Ales Musil
+feature_name: oVirt LLDP labeler
+feature_modules: ovirt-lldp-labeler
+feature_status: In Development
+---
+
+# oVirt LLDP labeler
+
+## Summary
+
+oVirt LLDP labeler is a tool that utilizes [LLDP support](https://ovirt.org/blog/2017/11/webadmin-lldp/) 
+added to oVirt in 4.2. The tool helps to some extent automate network 
+configuration overhead in clusters that are managed by the labeler. 
+It can be used as a separate service or as one-time tool using its cli.
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+
+### Benefit to oVirt
+
+The labeler is service that is capable of helping oVirt administrator
+to automate some network related tasks. According to configuration reported 
+via LLDP from peer switch, the labeler is capable of labeling host network 
+interface or bond interfaces together. 
+
+### Typical user flow
+
+Consider following scenario:
+
+Engine contains logical network `vlan10` tagged as VLAN with id 10
+and label `lldp_vlan_10`. Multiple hosts that have it's `eth1` 
+connected to switch with VLAN id 10 configured and LLDP reporting 
+enabled for the interfaces connected to the hosts. To automatically 
+connected our logical network `vlan10` with the `eth1`, on every host,
+the administrator just needs to configure and run the labeler:
+
+First time setup:
+
+1) Write username and password into `/etc/ovirt-lldp-labeler/conf.d/ovirt-lldp-credentials.conf`
+2) Write cluster name and API address into `/etc/ovirt-lldp-labeler/conf.d/ovirt-lldp-labeler.conf`
+
+Repeatable setup:
+
+1) Run the oVirt LLDP labeler by executing `python /usr/share/ovirt-lldp-labeler/ovirt_lldp_labeler_cli.py`
+  
+After the labeler run the hosts will automatically connect their
+`eth1` with `vlan10` network, due to matching label. This can be 
```

Comment:
Please write on which entities which label is attached, and that the labeler is doing this.